nf_wochenende.nf

```diff
@@ -62,13 +62,13 @@ params.metagenome = ""
 params.aligner = ""
 params.mismatches = ""
 params.nextera = ""
-params.abra = ""
+// params.abra = ""
 params.mapping_quality = ""
 params.readType = ""
 params.debug = ""
 params.longread = ""
-params.no_dup_removal = ""
-params.no_prinseq = ""
+// params.no_dup_removal = ""
+// params.no_prinseq = ""
 params.no_fastqc = ""
 params.fastp = ""
 params.trim_galore = ""
```

Comment:
The others like trim_galore, longread, debug, ... still need to be added, but I won't do that now. That can be done when everything else works and when we know, that we won't chenge the way this script is started. At least I think that it is not so important right now